airbyte-integrations/connectors/source-jira/integration_tests/full_configured_catalog.json

```diff
@@ -9534,10 +9534,10 @@
         },
         "supported_sync_modes": ["incremental"],
         "source_defined_cursor": true,
-        "default_cursor_field": ["startedAfter"]
+        "default_cursor_field": ["updated"]
```

Comment:
same question

Reply:
Yes I checked and saw it in the record, here is the example:

`{"type": "RECORD", "record": {"stream": "issues", "data": {"expand": "operations,versionedRepresentations,editmeta,changelog,renderedFields", "id": "10026", "self": "https://airbyteio.atlassian.net/rest/api/3/issue/10026", "key": "EX-1", "fields": {"issuetype": {"self": "https://airbyteio.atlassian.net/rest/api/3/issuetype/10004", "id": "10004", "description": "A problem or error.", "iconUrl": "https://airbyteio.atlassian.net/secure/viewavatar?size=medium&avatarId=10303&avatarType=issuetype", "name": "Bug", "subtask": false, "avatarId": 10303, "hierarchyLevel": 0}, "security": null, "issuelinks": [{"id": "10262", "self": "https://airbyteio.atlassian.net/rest/api/3/issueLink/10262", "type": {"id": "10002", "name": "Duplicate", "inward": "is duplicated by", "outward": "duplicates", "self": "https://airbyteio.atlassian.net/rest/api/3/issueLinkType/10002"}, "outwardIssue": {"id": "10031", "key": "EX-2", "self": "https://airbyteio.atlassian.net/rest/api/3/issue/10031", "fields": {"summary": "Test 25", "status": {"self": "https://airbyteio.atlassian.net/rest/api/3/status/10000", "description": "", "iconUrl": "https://airbyteio.atlassian.net/", "name": "To Do", "id": "10000", "statusCategory": {"self": "https://airbyteio.atlassian.net/rest/api/3/statuscategory/2", "id": 2, "key": "new", "colorName": "blue-gray", "name": "To Do"}}, "priority": {"self": "https://airbyteio.atlassian.net/rest/api/3/priority/3", "iconUrl": "https://airbyteio.atlassian.net/images/icons/priorities/medium.svg", "name": "Medium", "id": "3"}, "issuetype": {"self": "https://airbyteio.atlassian.net/rest/api/3/issuetype/10001", "id": "10001", "description": "Functionality or a feature expressed as a user goal.", "iconUrl": "https://airbyteio.atlassian.net/secure/viewavatar?size=medium&avatarId=10315&avatarType=issuetype", "name": "Story", "subtask": false, "avatarId": 10315, "hierarchyLevel": 0}}}}], "attachment": [{"self": "https://airbyteio.atlassian.net/rest/api/3/attachment/10212", "id": "10212", "filename": "demo.xls", "author": {"self": "https://airbyteio.atlassian.net/rest/api/3/user?accountId=5fc9e78d2730d800760becc4", "accountId": "5fc9e78d2730d800760becc4", "emailAddress": "integration-test@airbyte.io", "avatarUrls": {"48x48": "https://secure.gravatar.com/avatar/0a7841feac7218131ce7b427283c24ef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FIT-5.png", "24x24": "https://secure.gravatar.com/avatar/0a7841feac7218131ce7b427283c24ef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FIT-5.png", "16x16": "https://secure.gravatar.com/avatar/0a7841feac7218131ce7b427283c24ef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FIT-5.png", "32x32": "https://secure.gravatar.com/avatar/0a7841feac7218131ce7b427283c24ef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FIT-5.png"}, "displayName": "integration test", "active": true, "timeZone": "America/Los_Angeles", "accountType": "atlassian"}, "created": "2021-04-14T14:14:47.369-0700", "size": 13824, "content": "https://airbyteio.atlassian.net/secure/attachment/10212/demo.xls"}, {"self": "https://airbyteio.atlassian.net/rest/api/3/attachment/10213", "id": "10213", "filename": "demo.xlsx", "author": {"self": "https://airbyteio.atlassian.net/rest/api/3/user?accountId=5fc9e78d2730d800760becc4", "accountId": "5fc9e78d2730d800760becc4", "emailAddress": "integration-test@airbyte.io", "avatarUrls": {"48x48": "https://secure.gravatar.com/avatar/0a7841feac7218131ce7b427283c24ef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FIT-5.png", "24x24": "https://secure.gravatar.com/avatar/0a7841feac7218131ce7b427283c24ef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FIT-5.png", "16x16": "https://secure.gravatar.com/avatar/0a7841feac7218131ce7b427283c24ef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FIT-5.png", "32x32": "https://secure.gravatar.com/avatar/0a7841feac7218131ce7b427283c24ef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FIT-5.png"}, "displayName": "integration test", "active": true, "timeZone": "America/Los_Angeles", "accountType": "atlassian"}, "created": "2021-04-14T14:14:48.276-0700", "size": 7360, "content": "https://airbyteio.atlassian.net/secure/attachment/10213/demo.xlsx"}], "updated": "2021-04-15T11:40:58.036-0700"}}, "emitted_at": 1626340244000}}`

Reply:
@po3na4skld  but we talk about Issue worklogs. I'm not sure if we can filter by thus [field](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-worklogs/#api-rest-api-3-issue-issueidorkey-worklog-get).